frontend/src/routes/Governance/common/util.tsx

```diff
@@ -484,7 +484,7 @@ export function getPolicyRemediation(policy: Policy | undefined, propagatedPolic
   const remediationSet = new Set()
 
   templates?.forEach((template: PolicyTemplate) => {
-    const templateRemediation = template.objectDefinition.spec.remediationAction
+    const templateRemediation = template.objectDefinition.spec?.remediationAction ?? 'inform'
```

Comment:
Default is inform

Reply:
Should the template be ignored instead if `remediationAction` isn't set? Otherwise, it won't hit this code block further down:
```
    // empty root remediation and template remedation all empty
    if (!rootRA) {
```

Reply:
because of informOnly we should consider template remediation when root is set enforce or inform

Reply:
I guess what I'm poking at is that in the issue, a policy type isn't even provided and it's missing the `spec` also. I feel like if the `spec` is missing AND it's not even a policy type, the remediationAction is ignored. Otherwise, with the policy in the issue, when the root remediation isn't defined the UI would still show `inform`, which isn't accurate.

Reply:
OH now I get it  Thanks!